prow/pubsub/subscriber/subscriber.go

```diff
@@ -478,6 +478,12 @@ func tryGetCloneURIAndHost(pe ProwJobEvent) (cloneURI, host string) {
 		return "", ""
 	}
 
+	// If the Refs struct already has a populated CloneURI field, use that
+	// instead.
+	if refs.CloneURI != "" {
```

Comment:
No integration test, but we will have an integration test for it after I update https://github.com/kubernetes/test-infra/pull/26563. I suppose I can add one now because I don't think one was added originally for this function.

Reply:
sorry I meant unit test, I'll not ask you to add one if not present

Reply:
Done